app/views/posts/index.html.erb

```diff
@@ -3,9 +3,27 @@
   <head>
     <title>GApp</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="application.css">
   </head>
   <body>
-    <h1>List of all posts</h1>
-<p>Placeholder: Here is a list of posts for a given user.</p>
- </body>
+  <h1>List of Posts</h1>
+
+<div class="post-list">
+  <% @posts.each do |post| %>
+    <div class="container">
+      <h2><%= post.title %></h2>
+      <p><%= post.text %></p>
+      <div class="comments">
+        <p><%= post.comments.count %> Comments</p>
+        <ul>
+          <% post.comments.each do |comment| %>
+            <li><%= comment.text %></li>
+          <% end %>
```

Comment:
- [ ] (Optional) It would be nice if by clicking on each post, a user would be taken to the user post show page.